model/project.go

```diff
@@ -40,6 +40,8 @@ type GetProjectTasksOpts struct {
 	Limit        int    `json:"num_versions"`
 	BuildVariant string `json:"build_variant"`
 	StartAt      int    `json:"start_at"`
+	PatchOnly    bool   `json:"patch_only"`
```

Comment:
I'd prefer if this took in a list of requesters since that's more consistent with other routes, and is more configurable (and it should probably default to just repotracker versions

Reply:
updated